sdk/identity/Azure.Identity/src/LinuxVisualStudioCodeAdapter.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+using Azure.Core;
+
+namespace Azure.Identity
+{
+    internal sealed class LinuxVisualStudioCodeAdapter : IVisualStudioCodeAdapter
+    {
+        private static readonly string s_userSettingsJsonPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "Code", "User", "settings.json");
```

Comment:
Seem like settings on Linux is under $HOME/.config/Code/User/settings.json.

See https://vscode.readthedocs.io/en/latest/getstarted/settings/